lib/xcflushd/flusher.rb

```diff
@@ -9,12 +9,13 @@ class Flusher
 
     XcflushdError = Class.new(StandardError)
 
-    def initialize(reporter, authorizer, storage, auth_ttl, error_handler, threads)
+    def initialize(reporter, authorizer, storage, auth_ttl, error_handler, logger, threads)
```

Comment:
Agreed, and there are more classes with the same problem. Let's fix that in a separate PR.